Showing higher audience count in destination than the Segment UI. (#4900)
segmentio/segment-docs: src/engage/faqs.md

## Can I reuse audience keys?

> Avoid using the same audience key twice, even if you've deleted the key's original audience. Downstream tools and Destinations might have trouble distinguishing between different audiences that at any point shared the same key. And this may create mismatch on the audience size between Segment and Destination as the destination may have more audience count as the users of the old audience still exist over the destination.

Reviewer: ```suggestion
Avoid using the same audience key twice, even if you've deleted the key's original audience. Downstream tools and destinations might have trouble distinguishing between different audiences that once shared the same key. This may create mismatch in audience size between Segment and the destination because the destination may count users of the old audience, resulting in a larger audience size.
```
